### ISSUE TITLE
bug: MD054 incorrectly flags task lists when `shortcut = false`

### ISSUE THREAD

**Issue by martimlobao** [rvben/rumdl#221]
False positive for MD054 when detecting shortcut links.

```toml
[global]
enable = ["MD054"]

[MD054]
autolink = true    # Allow <https://example.com>
inline = true      # Allow [text](url) and ![alt](url)
full = true        # Allow [text][ref] with separate [ref]: url definition
collapsed = true   # Allow [text][] with separate [text]: url definition
shortcut = false    # Allow [text] with separate [text]: url definition
url-inline = true  # Allow [https://example.com](https://example.com)
```

```md
# Todos

- [ ] Task 1
```

```
❯ rumdl check todo.md
todo.md:3:3: [MD054] Link/image style 'shortcut' is not allowed
```

**Comment by rvben** [comment]
Fixed in v0.0.200. Thanks for reporting @martimlobao!
